components/vocabularies/hgnc/ui/src/main/resources/PhenoTips/GeneNameService.xml

```diff
@@ -57,6 +57,7 @@
       #set ($term = $services.vocabularies.hgnc.getTerm($i))
       #if ($term)
         #set ($result = {
+          'id' : $term.underlyingId,
```

Comment:
Since we're manually building this, you can just use `'id': $stringutils.defaultIfBlank($term.ensembl_gene_id, $term.symbol)` and get rid of the `getUnderlyingId` method completely.